From: Leon Romanovsky <leonro@mellanox.com>
Date: Wed, 21 Feb 2018 18:12:42 +0200
Subject: RDMA/uverbs: Replace user's types with kernel's types
Patch-mainline: v4.17-rc1
Git-commit: cd35cf4b40f351fc9e53c2e2877a56da87dcd46d
References: bsc#1103992 FATE#326009

The internal to kernel variable declarations don't need to be
declared with user types. This patch converts such occurrences
appeared in ib_uverbs_write().

Signed-off-by: Leon Romanovsky <leonro@mellanox.com>
Signed-off-by: Doug Ledford <dledford@redhat.com>
Acked-by: Thomas Bogendoerfer <tbogendoerfer@suse.de>
---
 drivers/infiniband/core/uverbs_main.c |   10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

--- a/drivers/infiniband/core/uverbs_main.c
+++ b/drivers/infiniband/core/uverbs_main.c
@@ -636,7 +636,7 @@ err_put_refs:
 }
 
 static bool verify_command_mask(struct ib_device *ib_dev,
-				__u32 command, bool extended)
+				u32 command, bool extended)
 {
 	if (!extended)
 		return ib_dev->uverbs_cmd_mask & BIT_ULL(command);
@@ -655,11 +655,11 @@ static bool verify_command_idx(u32 comma
 }
 
 static ssize_t process_hdr(struct ib_uverbs_cmd_hdr *hdr,
-			   __u32 *command, bool *extended)
+			   u32 *command, bool *extended)
 {
-	__u32 flags;
+	u32 flags;
 
-	if (hdr->command & ~(__u32)(IB_USER_VERBS_CMD_FLAGS_MASK |
+	if (hdr->command & ~(u32)(IB_USER_VERBS_CMD_FLAGS_MASK |
 				   IB_USER_VERBS_CMD_COMMAND_MASK))
 		return -EINVAL;
 
@@ -721,8 +721,8 @@ static ssize_t ib_uverbs_write(struct fi
 	struct ib_device *ib_dev;
 	struct ib_uverbs_cmd_hdr hdr;
 	bool extended;
-	__u32 command;
 	int srcu_key;
+	u32 command;
 	ssize_t ret;
 
 	if (!ib_safe_file_access(filp)) {
